src/pretix/presale/style.py

```diff
@@ -312,8 +167,53 @@ def get_font_stylesheet(font_name, event: Event = None):
                 if formats[f].startswith('https'):
                     srcs.append(f"url('{formats[f]}') format('{f}')")
                 else:
-                    srcs.append(f"url(static('{formats[f]}')) format('{f}')")
+                    srcs.append(f"url('{static(formats[f])}') format('{f}')")
         stylesheet.append("src: {};".format(", ".join(srcs)))
         stylesheet.append("font-display: swap;")
         stylesheet.append("}")
     return "\n".join(stylesheet)
+
+
+def get_theme_vars_css(obj, widget=False):
+    sassrules = []
+    if obj.settings.get("primary_color"):
+        sassrules.append("$in-brand-primary: {};".format(obj.settings.get("primary_color")))
+    if obj.settings.get("theme_color_success"):
+        sassrules.append("$in-brand-success: {};".format(obj.settings.get("theme_color_success")))
+    if obj.settings.get("theme_color_danger"):
+        sassrules.append("$in-brand-danger: {};".format(obj.settings.get("theme_color_danger")))
+    if obj.settings.get("theme_color_background"):
+        sassrules.append("$in-body-bg: {};".format(obj.settings.get("theme_color_background")))
+    if not obj.settings.get("theme_round_borders"):
+        sassrules.append("$in-border-radius-base: 0;")
+        sassrules.append("$in-border-radius-large: 0;")
+        sassrules.append("$in-border-radius-small: 0;")
+
+    font = obj.settings.get("primary_font")
+    if font != "Open Sans" and not widget:
+        sassrules.append(get_font_stylesheet(
+            font,
+            event=obj if isinstance(obj, Event) else None,
+            organizer=obj.organizer if isinstance(obj, Event) else obj,
+            absolute=False,
+        ))
+        sassrules.append(
+            '$in-font-family-sans-serif: "{}", "Open Sans", "OpenSans", "Helvetica Neue", Helvetica, Arial, sans-serif '
+            '!default'.format(
+                font
+            )
+        )
+
+    if widget:
+        sassrules.append("$widget: true;")
+
+    with open(finders.find("pretixbase/scss/_variables.scss"), "r") as f:
+        source_scss = f.read()
+        sassrules.append(source_scss)
+
+    sassdir = os.path.join(settings.STATIC_ROOT, "pretixbase/scss")
+    css = sass.compile(
+        string="\n".join(sassrules),
+        include_paths=[sassdir]
+    )
```

Comment:
ist sass.compile performant genug, dass wir das direkt beim rendern der response so machen sollten? (ich vermute die antwort ist ja, kurzes testen hat ergeben dass es auf meinem laptop ca 10ms unterschied ausmacht wenn ich diesen block durch ein `css = "\n".join(sassrules)` ersetze)

ich frage mich, brauchen wir an dieser stelle überhaupt noch sass? wir machen ja quasi nur noch template-ersetzung mit einer handvoll farb-berechnungen. das könnte man ja ggf auch mit einem jinja2-template lösen. aber solange es keine probleme verursacht können wir auch einfach bei sass bleiben.

Reply:
Yeah I had the same question and the same experimental result. We could probably do it in a Django template but I did not quickly find a library that does all of the color computations efficiently. libsass is native code, so a python implementation of the color computations is even likely to be slower. We would save on the parsing and import time though

Reply:
Maybe some time in the future, we could swap to [CSS color-mix](https://developer.mozilla.org/en-US/blog/color-palettes-css-color-mix/)? Then no pre/post-processor would be needed. Although to have this somewhat backwards-compatible would mean the same effort to have this PR with `var()` colors compatible with IE – ie. doubling all color-related properties with the unmixed and mixed color.

Reply:
Oh I did not notice color-mix being well supported now, I looked for stuff like the lighten and the contrasts and the functions I found were not even in recent Firefox yet